assets/js/modules/analytics/common/create-account-field.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * CreateAccountField component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	Input,
+	TextField,
+} from '../../../material-components';
+import classnames from 'classnames';
+
+const CreateAccountField = ( {
+	validationIssues,
+	setValidationIssues,
+	objectValue,
+	objectSetter,
+	name,
+	label,
+} ) => {
+	return (
+		<TextField
+			className={ classnames(
+				'mdc-text-field',
+				{ 'mdc-text-field--error': validationIssues[ name ] }
```

Comment:
Similarly, let's simplify this prop to be more targeted, something like `hasError`. The `validationIssues` part can then remain internal to the main component.

Reply:
update with `hasError` in https://github.com/google/site-kit-wp/pull/1389/commits/527201e0a64fca7b4eae50a402147fe29fd0417f.